### ISSUE TITLE
NO_SQL_DATA error returned on CREATE TABLE and others

### ISSUE THREAD

**Issue by kskoeld** [JuliaDatabases/ODBC.jl#236]
I keep getting errors using AWS Athena ODBS driver ( and earlier Redshift) whenever I send Queries with `CREATE TABLE` `CREATE VIEW` `DROP TABLE` and the like.
Naturally such queries do not return data,  so the error message doesn't really make sense here.
Any known workaround for this?
I tried handling these situation with 
```
        try
            df=ODBC.query(dsn,strSQL)
        catch err
            if occursin(err.msg,"SQL_NO_DATA")
                println("done")
            else
                println(err)
                println(strSQL)
                return err
            end

        end
```
but that doesn't work.

**Comment by pnorth1** [comment]
For these transaction statements it should work to use `ODBC.execute!` instead of `ODBC.query`

**Comment by quinnj** [comment]
This changed in a recent release; before we were swallowing errors when fetching data, but obviously `NO_SQL_DATA` is a valid return code here. I'll work on a fix.

**Comment by quinnj** [comment]
Should be fixed with #263
